tn-cli/sample-macro-script.txt

```diff
@@ -0,0 +1,46 @@
+# This script shows what can be done with tn-cli macros. Run it as
+#
+#   python tn-cli.py --no-login < sample-macro-script.txt
+#
+# The script performs the following:
+#
+#  - Logs in as Xena (user has root privileges)
+#  - Creates a new user Test User with useradd
+#  - Changes Test User's name to 'Test User 1'
+#  - Modifies Test User's default anon acs to JR with chacs
+#  - Sets Test User's password to test456
+#  - Deletes Test User with userdel
+#  - Uses resolve to locate user Alice
+#  - Subscribes to Alice
+#  - Sends a message to Alice
+
+# Create user 'Test User'
```

Comment:
Maybe add an explanation that `.must` stops execution if command fails and and `$user` is a variable which will hold the result of the execution.

Reply:
Done.